### ISSUE TITLE
MixInstructions dictionary isnt supporting more than 1 mixing instruction (pls close, solved already)

### ISSUE THREAD

**Issue by LaCumbiaDelCoronavirus** [mestiez/ppg-snippets#45]
(bug report)

/ I already solved this issue, please close! /

Solution: Tried something random, have no idea what fixed it.

I am trying to make a script that adds multiple liquids, and ofcourse, 4 liquid mixers to the game.

For more clarity:
This is my code for each recipe (I have not misspelt the liquids, and they are all registered in the liquid registry).
I have changed the names of the most of the variables for simplicity

(repeated 3 more times with values changed)

```
// Recipe1

var mixer1 = new LiquidMixInstructions(
     Liquid.GetLiquid(liquid1.ID), // (this is changed)
     Liquid.GetLiquid(liquid2.ID), // (this is changed)
     Liquid.GetLiquid(targetliquid.ID)); // (this is changed)

mixer1.RatePerSecond = 0.4f; // rate of creation (liquid units per second) (this is changed)

// prevent liquids from mixing in limbs (this is not changed)
mixer1.ContainerFilter = container => !(container is CirculationBehaviour);
```

and here is the code that adds the recipes to the mixinstructions dictionary (after all the recipes, not before)

```
LiquidMixingController.MixInstructions.Add(mixer1);
LiquidMixingController.MixInstructions.Add(mixer2);
LiquidMixingController.MixInstructions.Add(mixer3);
LiquidMixingController.MixInstructions.Add(mixer4);
```

When testing in game, all parts of the mod work fine, no errors. But, only the first recipe works...
More specifically, it is the first recipe to be created/defined, not the first recipe to be added to the mixinstructions dictionary
The documentation says nothing about this.

So that some time isnt wasted:
I mentioned previously that nothing is misspelt and that every liquid is in the liquid registry. None of the liquid IDs of each liquid are the same as any other liquid, and have their own mod tag, so they cant be mixed up with pre-existing liquids in the game. All the liquids and mixing recipes have been added in the Main method.

If this is something to do with the documentation, please make it more apparent, or is it about the game only checking the mixing recipe dictionary with the first recipe and ignoring all the rest?

thankz

TL:DR
Only 1 mixing instruction works, but all the rest are ignored
